pennylane/templates/broadcast.py

```diff
@@ -82,41 +105,16 @@ def _preprocess(parameters, pattern, wires):
         wire_sequence, parameters: preprocessed pattern and parameters
     """
 
-    custom_pattern = None
-
     if isinstance(pattern, str):
+        _wires = wires
         if pattern not in OPTIONS:
             raise ValueError(f"did not recognize pattern {pattern}".format())
     else:
         # turn custom pattern into list of Wires objects
-        custom_pattern = [Wires(w) for w in pattern]
+        _wires = [Wires(w) for w in pattern]
```

Comment:
Actually, in the new approach of being stingy with Wires parsing we could just remove this line altogether. This was in some sense just a check if the pattern has unique wires in every set. But if not, the gate will later complain, so this should be fine!

Reply:
so instead just set `_wires = pattern`?